pkg/distro/BUILD

```diff
@@ -1,14 +1,16 @@
-package(
-    default_visibility = ["//visibility:private"],
-)
-
 load("//:pkg.bzl", "pkg_tar")
 load("//:version.bzl", "version")
 load("//releasing:defs.bzl", "print_rel_notes")
 load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
 load("@bazel_stardoc//stardoc:stardoc.bzl", "stardoc")
 load("@rules_python//python:defs.bzl", "py_test")
 
+licenses(["notice"])
+
+package(
+    default_visibility = ["//visibility:private"],
+)
```

Comment:
nit: could probably throw this out if you wanted to

Reply:
I think I want it private. The code should not be able to depend on the rules which build the distribution from the code which depends on the rules which build the distribution from the code which ....

Reply:
Hah. I totally agree that it should be private here; I was more commenting on how it's redundant, since `default_visibility` defaults to `//visibility:private`.